packages/trpc-interface/src/shared/client.ts

```diff
@@ -0,0 +1,37 @@
+import { createTRPCProxyClient, httpBatchLink } from "@trpc/client";
+import { sharedRouter, type SharedRouter } from "./router";
+import { callerLink } from "../trpc-caller-link";
+
+type SharedClientOptions = {
+  url: string;
+  apiToken: string;
+};
+
+export const createTRPCProxyServiceClient = (
```

Comment:
lets follow our convention for our code base, despite that trpc uses a different one
```suggestion
export const createTrpcProxyServiceClient = (
```